bottlerocket-settings-models/settings-extensions/motd/Cargo.toml

```diff
@@ -7,11 +7,7 @@ edition = "2021"
 publish = false
 
 [dependencies]
```

Comment:
`motd` setting extension still needs `env_logger`, worth doing in this commit so we don't forget?